src/lib/utils/balancer/pool.ts

```diff
@@ -0,0 +1,5 @@
+import { getAddress } from 'ethers/lib/utils';
+
+export function getPoolAddress(poolId: string) {
```

Comment:
Where is this used? I have since added the `address` property to the pool object returned from the subgraph so it might not be needed?